Kitodo/src/main/resources/hibernate.cfg.xml

```diff
@@ -37,6 +37,9 @@
         <property name="hibernate.connection.verifyServerCertificate">false</property>
         <property name="hibernate.connection.useSSL">false</property>
 
+        <!-- Hibernate search -->
```

Comment:
Are here the other Hibernate Search parameters are missing like used URI, port, ... which are added in the `hibernate.properties` file?

Reply:
I can confirm that I also notice the similarity. Needs testing.

Reply:
It would be nice if the Hibernate-Search properties are stored in one place / file. If this is not possible it would be bad at least for me.